packages/server/database/migrations/20180206135107-renameProjectToTask.js

```diff
@@ -1,24 +1,13 @@
 exports.up = async (r) => {
   // rename the Project/ProjectHistory tables
-  await r
-    .table('Project')
-    .config()
-    .update({name: 'Task'})
-    .run()
-  await r
-    .table('ProjectHistory')
-    .config()
-    .update({name: 'TaskHistory'})
-    .run()
-  await r
-    .table('TaskHistory')
-    .indexDrop('projectIdUpdatedAt')
-    .run()
+  await r.table('Project').config().update({name: 'Task'}).run()
+  await r.table('Task').wait()
```

Comment:
moved this line up a few to give it a few more ms to kick in.
this shouldn't be necessary, but we live in a crazy world.